YOZM/YOZM/View/Component/StudyCardBackground.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  StudyCardBackground.swift
+//  YOZM
+//
+//  Created by 정희균 on 8/29/25.
+//
+
+import SwiftUI
+
+struct StudyCardBackground: View {
+    var body: some View {
+        RoundedRectangle(cornerRadius: 16)
+            .fill(Color.yellowWhite)
+            .shadow(
+                color: Color(hex: "646464").opacity(0.25),
+                radius: 0,
+                x: 2,
+                y: 6
+            )
```

Comment:
Color Extension으로 문자열을 받아서 처리하고 있던데 Asset에 등록된 Color를 그대로 사용하지 않은 이유가 있을까여??


아래와 같은 방식으로요!

```suggestion
            .shadow(
                color: Color(.BlueDark).opacity(0.25),
                radius: 0,
                x: 2,
                y: 6
            )
```

Reply:
타키가 올려놓은 컬러 시스템이랑 실제 디자인에 적용된 컬러가 달라서 해당 뷰에서만 사용하는 컬러는 뷰에서 초기화하도록 했어요!

Reply:
아하 확인했습니다!